docs/static/winlogbeat-modules.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[winlogbeat-modules]]
+
+== Working with {winlogbeat} Modules
+
+{winlogbeat} comes packaged with pre-built
+{winlogbeat-ref}/winlogbeat-modules.html[modules] that contain the configurations
+needed to collect, parse, enrich, and visualize data from various Windows logging
+providers. Each {winlogbeat} module consists of one or more filesets that contain
+ingest node pipelines, {es} templates, {winlogbeat} input configurations, and
+{kib} dashboards.
+
+You can use {winlogbeat} modules with {ls}, but you need to do some extra setup.
+The simplest approach is to <<use-ingest-pipelines,set up and use the ingest
+pipelines>> provided by {winlogbeat}.
+
+[[use-ingest-pipelines]]
```

Comment:
You might need to add a `[float]` or `[discrete]` tag here if you want to keep all the content together on one page. (Just look at the output--sorry I don't have time because I am technically on PTO. :-) 

@karenzone FYI that this PR might need some care and feeding.